### ISSUE TITLE
Make a way to display local IP when server start 

### ISSUE THREAD

**Issue by lucastucious** [nwesterhausen/valheim-discordconnector#210]
Similary to the %PUBLICIP% variable we had, can we have a %LOCALIP% that display lan IP (192.168.0.x) ?

**Comment by nwesterhausen** [comment]
You have an option to set one of the message variables to be something to indicate this. For example on one of the servers you can set `%VAR1%` to say "LAN Server" and on the other server set it to "Crossplay Server". 

Is it the case that you use the exact same configurations of everything and only change the launch message?

I think there probably is a way to determine if crossplay is enabled which I could add, but I have to figure that out.

**Comment by lucastucious** [comment]
Yes that's the same server, same files, just the start_server.bat have a
different commandline.

So you say I can set a %var% in this command line ?

**Comment by nwesterhausen** [comment]
No sorry for confusion, you cannot set that in command line. I will see what I can do about adding something like `%CROSSPLAY%` to be something that displays one thing when crossplay and something else if not (both set by your config).

The way I'm determining the IP address is from the game itself, asking it what its public IP is. I don't want to fiddle around and detect something on my own if I don't need to.
